packages/devtools_app/lib/src/screens/deep_link_validation/validation_details_view.dart

```diff
@@ -450,75 +450,115 @@ class _DomainAssociatedLinksPanel extends StatelessWidget {
+            title: const Row(
+              children: [
+                SizedBox(width: defaultSpacing),
+                Expanded(
+                  child: Text('OS'),
+                ),
+                Expanded(
+                  child: Text('Issue type'),
+                ),
+                Expanded(
+                  child: Text(
+                    'Status',
+                  ),
+                ),
+              ],
             ),
-            dataRowColor: MaterialStateProperty.all(
-              theme.colorScheme.alternatingBackgroundColor2,
+          ),
+          const Divider(height: 1.0),
+          ExpansionTile(
+            backgroundColor:
+                Theme.of(context).colorScheme.alternatingBackgroundColor2,
+            collapsedBackgroundColor:
+                Theme.of(context).colorScheme.alternatingBackgroundColor2,
+            title: Row(
+              children: [
+                const SizedBox(width: defaultSpacing),
+                const Expanded(child: Text('Android')),
+                const Expanded(child: Text('IntentFiler')),
+                Expanded(
+                  child: intentFilterErrorCount > 0
+                      ? Text(
+                          '$intentFilterErrorCount Check failed',
+                          style: TextStyle(
+                            color: Theme.of(context).colorScheme.error,
+                          ),
+                        )
+                      : noIssueText,
+                ),
+              ],
             ),
-            columns: const [
-              DataColumn(label: Text('OS')),
-              DataColumn(label: Text('Issue type')),
-              DataColumn(label: Text('Status')),
+            children: <Widget>[
+              for (final error in intentFilterErrors)
+                if (linkData.pathErrors.contains(error))
+                  Text(error.description),
             ],
-            rows: [
-              DataRow(
-                cells: [
-                  const DataCell(Text('Android')),
-                  const DataCell(Text('Intent filter')),
-                  notAvailableCell,
-                ],
-              ),
-              DataRow(
-                cells: [
-                  const DataCell(Text('iOS')),
-                  const DataCell(Text('Associated domain')),
-                  notAvailableCell,
-                ],
-              ),
-              DataRow(
-                cells: [
-                  const DataCell(Text('Android, iOS')),
-                  const DataCell(Text('URL format')),
-                  notAvailableCell,
-                ],
-              ),
-              DataRow(
-                cells: [
-                  const DataCell(Text('Android, iOS')),
-                  const DataCell(Text('Routing')),
-                  notAvailableCell,
-                ],
-              ),
+          ),
+          const Divider(height: 1.0),
+          ExpansionTile(
```

Comment:
much of this widget is duplicated above. Can you make a generalized helper widget for use in both places?